frontend/packages/kubevirt-plugin/src/components/modals/clone-vm-modal/configuration-summary.tsx

```diff
@@ -42,11 +43,13 @@ const getDisksDescription = (
     const volume = volumes.find((v) => v.name === disk.name);
     if (volume) {
       if (volume.dataVolume) {
-        let dataVolume = dataVolumeTemplates.find((dv) => getName(dv) === volume.dataVolume.name);
+        let dataVolume = dataVolumeTemplates.find(
+          (dv) => getName(dv) === volume.dataVolume.name,
+        ) as V1alpha1DataVolume;
```

Comment:
you dont have to typecast here and anywhere else. Just specify the correct types from the beginning

Reply:
fixed :+1: